src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/afcalc/GeneralPloidyExactAFCalculator.java

```diff
@@ -355,396 +310,12 @@ private double computeLofK(final ExactACset set,
-            // for each entry in logPL table, associated originally with allele count stored in vec[],
-            // see if this allele count conformation will be present in new logPL table.
-            // For entry to be present, elements in dimensions not present in requested allele list have to have count = 0
-            final int[] pVec = iterator.getCurrentVector();
-            final double pl = oldLikelihoods[iterator.getLinearIndex()];
-
-            boolean keyPresent = true;
-            for (int k=0; k < allelePresent.length; k++) {
-                if (pVec[k] > 0 && !allelePresent[k]) {
-                    keyPresent = false;
-                }
-            }
-
-            if (keyPresent) {// skip to next entry in logPLs if this conformation is not present in subset
-
-                final int[] newCount = new int[allelesToSubset.size()];
-
-                // map from old allele mapping count to new allele mapping
-                // In pseudo-Matlab notation: newCount = vec[permutationKey] for permutationKey vector
-                for (idx = 0; idx < newCount.length; idx++) {
-                    newCount[idx] = pVec[permutationKey[idx]];
-                }
-
-                // get corresponding index from new count
-                final int outputIdx = getLinearIndex(newCount, allelesToSubset.size(), numChromosomes);
-                newPLs[outputIdx] = pl;
-            }
-            iterator.next();
-        }
-
-        return  newPLs;
-    }
-
-    private static int getLinearIndex(final int[] vectorIdx, final int numAlleles, final int ploidy) {
-
-        if (ploidy <= 0) {
-            return 0;
-        }
-
-        int linearIdx = 0;
-        int cumSum = ploidy;
-        for (int k=numAlleles-1;k>=1; k--) {
-            final int idx = vectorIdx[k];
-            // how many blocks are before current position
-            if (idx == 0) {
-                continue;
-            }
-            for (int p=0; p < idx; p++) {
-                linearIdx += getNumLikelihoodElements(k, cumSum - p);
-            }
-
-            cumSum -= idx;
-        }
-
-        return linearIdx;
-
-    }
-
-    /**
-     * Given a scalar index, what's the alelle count conformation corresponding to it?
-     * @param nAlleles                    Number of alleles
-     * @param numChromosomes              Ploidy
-     * @param PLindex                     Index to query
-     * @return                            Allele count conformation, according to iteration order from SumIterator
-     */
-    private int[] getAlleleCountFromPLIndex(final int nAlleles, final int numChromosomes, final int PLindex) {
-        final GenotypeLikelihoodCalculator calculator = calculators.getInstance(numChromosomes, nAlleles);
-        final GenotypeAlleleCounts alleleCounts = calculator.genotypeAlleleCountsAt(PLindex);
-        return alleleCounts.alleleCountsByIndex(nAlleles - 1);
+    //TODO: this probably belongs somewhere else
```

Comment:
After all the refactoring since I wrote this `TODO`, it seems like it actually doesn't really belong anywhere else.  It will be deleted when `GeneralPloidyExactAFCalculator` is discarded in the near future.